mlflow/tracking/fluent.py

```diff
@@ -1020,3 +1036,26 @@ def _get_experiment_id():
         or _get_experiment_id_from_env()
         or (is_in_databricks_notebook() and get_notebook_id())
     ) or deprecated_default_exp_id
+
+
+def universal_autolog(
+    log_input_example=False, log_model_signature=True
+):  # pylint: disable=unused-argument
+    # getargvalues isnt actually deprecated
+    # https://docs.python.org/3/library/inspect.html#inspect.getargvalues
+    arg_info = inspect.getargvalues(inspect.currentframe())  # pylint: disable=deprecated-method
```

Comment:
According to the docs for `inspect.currentframe()`, this may return `None` in certain environments (see https://docs.python.org/3/library/inspect.html#inspect.currentframe). Could we instead check explicitly for the presence of `log_input_example` and `log_model_signature` in the function signature inspected during `setup_autologging`?

Reply:
actually i don't think we need this.. we can just check locals() for any variables that each specific autolog integration might accept